packages/outbox-core/lib/outbox.ts

```diff
@@ -0,0 +1,170 @@
+import { AbstractPeriodicJob, type JobExecutionContext } from '@lokalise/background-jobs-common'
+import type { PeriodicJobDependencies } from '@lokalise/background-jobs-common/dist/periodic-jobs/periodicJobTypes'
+import { generateUuid7 } from '@lokalise/id-utils'
```

Comment:
considering that this is a public library, I'd suggest including uuid7 library as a direct dependency instead

Reply:
Sure!